SqlAlchemyTest1.py

```diff
@@ -1,58 +1,53 @@
-"""
-    Initializes a new instance of the Repository class.
-
-    Args:
-        logger: The logger object for logging.
-        session: The session object for database operations.
-        helper: The helper object for utility functions.
-"""
-
-from sqlalchemy import create_engine, Column, String, Integer
-from sqlalchemy.orm import sessionmaker
-from sqlalchemy.ext.declarative import declarative_base
-
-
-engine = create_engine('sqlite:///cousinsForTest.db', echo=True)
-
-Base = declarative_base()
-# comentario vazio
-# Define modelo da tabela
-class User(Base):
-    __tablename__ = 'users'
-
-    id = Column(Integer, primary_key=True)
-    name = Column(String)
-    age = Column(Integer)
-
-Base.metadata.create_all(engine)
-
-Session = sessionmaker(bind=engine)
+from database.database_utils import (
+    initialize_engine,
+    create_session,
+    create_tables,
+    add_users,
+    update_user_age,
+    delete_users,
+    query_users_ordered_by_age_split_by_18_years
+)
+from models.user import User
+
```

Comment:
Eu ainda encapsularia estas conexões com o banco e não deixaria no arquivo principal chamado para rodar o código. segue uns exemplos que peguei você pode usa ele.

`
from os import environ

from sqlalchemy import create_engine
from sqlalchemy.orm import sessionmaker


class Connection:

    def __init__(self, string_conn):
        self.engine = create_engine(string_conn)

    def get_engine(self):
        return self.engine

    def create_session(self):
        try:
            self.Session = sessionmaker(self.engine)
            self.session = self.Session()
            return self.session
        except Exception as error:
            return str(error)
`

Reply:
entao tipo, a ideia seria sempre que for fazer uma conexao usar a session e engine do objeto connection? e sempre que precisar de uma sessao nova (ou talvez uma engine nova, embora ainda nao saiba em que cenario isso fosse acontecer) fazer um novo objeto?